src/parca.py

```diff
@@ -84,27 +150,104 @@ def _fetch_version(self) -> str:
                 retries += 1
                 time.sleep(self._version_retry_wait)
 
-    def update_ca_certificate(self, ca_cert: str) -> None:
-        """Save the CA certificate file to disk and run update-ca-certificates."""
-        if self._container.can_connect():
-            current_ca_cert = (
-                self._container.pull(CA_CERT_PATH).read()
-                if self._container.exists(CA_CERT_PATH)
-                else ""
-            )
-            if current_ca_cert == ca_cert:
-                # No update needed
-                return
 
-            self._container.push(CA_CERT_PATH, ca_cert, make_dirs=True)
+def parca_command_line(
+    http_address: str = ":7070",
```

Comment:
Could we use the constant in the default here?
```suggestion
    http_address: str = f":{PARCA_PORT}",
```